modules/eventbridge/main.tf

```diff
@@ -0,0 +1,116 @@
+
+provider "aws" {
+  region = data.aws_region.current_region.name
+}
+locals {
```

Comment:
Should move to variables.

Reply:
its on purpose, i did it also in firehose, we have the `coralogix_region` variable, and then this local var inserts the relevant endpoint based on the region the customer put . see line 94